Failing tests:
> RuntimeError: quantized::linear (qnnpack): Expected input data type QUInt8 but got QInt8

See https://github.com/pytorch/pytorch/issues/199765

Author: Alexander Grund (TU Dresden)

diff --git a/test/quantization/fx/test_quantize_fx.py b/test/quantization/fx/test_quantize_fx.py
--- a/test/quantization/fx/test_quantize_fx.py
+++ b/test/quantization/fx/test_quantize_fx.py
@@ -5969,6 +5969,7 @@ class TestQuantizeFx(QuantizationTestCase):
             self._validate_qconfig_against_backend_config_constraints(
                 MyModel(), qconfig, backend_config, satisfies_constraints=True, qconfig_name=qconfig_name)
 
+    @unittest.skip("#199765")
     def test_symmetric_qnnpack_qconfig_mapping(self):
         """
         Test whether `torch.ao.quantization.qconfig_mapping._get_symmetric_qnnpack_qconfig_mapping`
@@ -6000,6 +6001,7 @@ class TestQuantizeFx(QuantizationTestCase):
             self.checkGraphModuleNodes(model, expected_node_occurrence=expected_node_occurrence)
             model(*example_inputs)
 
+    @unittest.skip("#199765")
     def test_symmetric_qnnpack_qat_qconfig_mapping(self):
         """
         Test whether `torch.ao.quantization.qconfig_mapping._get_symmetric_qnnpack_qat_qconfig_mapping`
